src/test/java/uk/gov/justice/hmpps/prison/executablespecification/BookingEventsStepDefinitions.java

```diff
@@ -45,7 +47,7 @@ public void responseEmpty() throws Throwable {
 
     @Then("^events are returned as follows:$")
     public void eventsAreReturnedAsFollows(final DataTable table) throws Throwable {
-        final var expected = table.asList(ScheduledEvent.class);
+        final List<ScheduledEvent> expected = table.asList(ScheduledEvent.class);
```

Comment:
need to specify the type so that `asList` knows what to cast it too

Reply:
weird that asList can't work out but I suppose that is an issue in DataTable